src/evo/simplifiedmns.cpp

```diff
@@ -187,19 +187,18 @@ bool BuildSimplifiedMNListDiff(const uint256& baseBlockHash, const uint256& bloc
 
     const CBlockIndex* baseBlockIndex = chainActive.Genesis();
     if (!baseBlockHash.IsNull()) {
-        auto it = mapBlockIndex.find(baseBlockHash);
-        if (it == mapBlockIndex.end()) {
+        baseBlockIndex = LookupBlockIndex(blockHash);
```

Comment:
```suggestion
        baseBlockIndex = LookupBlockIndex(baseBlockHash);
```

Fixes `feature_dip4_coinbasemerkleroots` failure. `feature_fee_estimation` still fails but didn't get the time yet to look into it.